modules/persistent-storage-csi-migration-overview.adoc

```diff
@@ -6,27 +6,6 @@
 [id="persistent-storage-csi-migration-overview_{context}"]
 = Overview
 
-With the automatic migration feature enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers.
+Volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart Container Storage Interface (CSI) drivers. This process does not involve any data migration, {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, nor is its contents changed.
```

Comment:
@lpettyjo could you add a section explaining how the default StorageClass is determined? Basically:

During TechPreview, the in-tree StorageClass is the default one. For GA:
1. The CSI SC will be the default in new 4.11 clusters.
2. The in-tree SC will remain the default one if the cluster was upgraded from 4.10. However, the CSI driver will be used under the hood (as opposed to the in-tree plugin).